File: src/main/java/dev/alexengrig/leetcode/_217_contains_duplicate/SortSolution.java
package dev.alexengrig.leetcode._217_contains_duplicate;

import java.util.Arrays;

class SortSolution implements Task {
    @Override
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        for (int i = 1; i < nums.length; i++) {
            if (nums[i - 1] == nums[i]) {
                return true;
            }
        }
        return false;
    }
}

File: src/main/python/217_contains_duplicate/sort_solution.py
from typing import List

from task import Task


class SortSolution(Task):
    def containsDuplicate(self, nums: List[int]) -> bool:
        nums.sort()
        for i in range(1, len(nums)):
            if nums[i - 1] == nums[i]:
                return True
        return False
